src/RadialGauge/RadialGaugeSeries/RadialGaugeSeries.tsx

```diff
@@ -76,42 +80,43 @@ export interface RadialGaugeSeriesProps {
   > | null;
 
   /**
-   * Min width for a guage. Only applicable in multi-series gauges.
+   * Min width for a gauge. Only applicable in multi-series gauges.
    */
   minGaugeWidth: number;
 }
 
-export class RadialGaugeSeries extends Component<RadialGaugeSeriesProps> {
-  static defaultProps: Partial<RadialGaugeSeriesProps> = {
-    outerArc: <RadialGaugeArc disabled={true} />,
-    innerArc: <RadialGaugeArc width={10} animated={true} />,
-    label: <RadialGaugeLabel />,
-    valueLabel: <RadialGaugeValueLabel />,
-    colorScheme: ['#00ECB1'],
-    padding: 10,
-    minGaugeWidth: 50
-  };
-
-  getWidths() {
-    const { data, width, height, minGaugeWidth } = this.props;
-
+function RadialGaugeSeries({
```

Comment:
Missing the `FC` type - I've been using this pattern:

```
export const RadialGaugeSeries: FC<Partial<RadialGaugeSeriesProps>> = ({
   myprop,
   ...rest
}) => {}
```

Reply:
Let's keep the export here please.

Reply:
moved the export up